docs/resources/ims_obs_system_image.md

```diff
@@ -0,0 +1,139 @@
+---
+subcategory: "Image Management Service (IMS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_ims_obs_system_image"
+description: |-
+  Manages an IMS system image resource created from external image file in the OBS bucket within HuaweiCloud.
+---
+
+# huaweicloud_ims_obs_system_image
+
+Manages an IMS system image resource created from external image file in the OBS bucket within HuaweiCloud.
+
+## Example Usage
+
+### Creating an IMS system image from an external image file in the OBS bucket
+
+```hcl
+variable "name" {}
+variable "image_url" {}
+variable "min_disk" {}
+
+resource "huaweicloud_ims_obs_system_image" "test" {
+  name      = var.name
+  image_url = var.image_url
+  min_disk  = var.min_disk
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the image.
+  The valid length is limited from `1` to `128` characters.
+  The first and last letters of the name cannot be spaces.
+  The name can contain uppercase letters, lowercase letters, numbers, spaces, chinese, and special characters (-._).
+
+* `image_url` - (Required, String, ForceNew) Specifies the URL of the external image file in the OBS bucket, the format
+  is **OBS bucket name:Image file name**. Changing this parameter will create a new resource.
+
+* `min_disk` - (Required, Int, ForceNew) Specifies the minimum size of the system disk, in GB unit.
+  Changing this parameter will create a new resource.
+  + When the operating system is Linux, the value ranges from `10` to `1,024`.
+  + When the operating system is Windows, the value ranges from `20` to `1,024`.
+
+* `description` - (Optional, String) Specifies the description of the image.
+
+* `type` - (Optional, String, ForceNew) Specifies the image type. The value can be **ECS**, **FusionCompute**, **BMS**,
+  or **Ironic**. Changing this parameter will create a new resource.
+  + Set to **ECS** or **FusionCompute** represent the creation of ECS server image.
+  + Set to **BMS** or **Ironic** represent the creation of BMS server image.
+
+* `is_config` - (Optional, Bool, ForceNew) Specifies whether to automatically configure. The value can be **true** or
+  **false**. Defaults to **false**. About the content of automatic backend configuration, please refer to
+  [API docs](https://support.huaweicloud.com/intl/en-us/ims_faq/ims_faq_0020.html).
+
```

Comment:
Changing this parameter will create a new resource.

Please check the other parts in documentation.